config/settings.yml

```diff
@@ -57,6 +57,7 @@ environment:
   label: "Beta"
   selector_name: "beta"
 current_cycle: 2021
```

Comment:
How painful is it to rename this to `current_recruitment_cycle`? I guess we can put it in a fixit friday ticket if it isn't straightforward

Reply:
This is best as a separate card I think, out of scope of this one.